CMakeLists.txt

```diff
@@ -29,6 +29,7 @@ set(SURGE_SKIP_VST3 TRUE CACHE BOOL "skip surge VST3 build")
 set(SURGE_SKIP_ALSA TRUE CACHE BOOL "skip surge Alsa build")
 set(SURGE_SKIP_STANDALONE TRUE CACHE BOOL "skip surge standalone build")
 set(SURGE_COMPILE_BLOCK_SIZE 8 CACHE STRING "surge compile block size")
+set(SURGE_SIMDE_PATH "${RACK_SDK_DIR}/dep/include" CACHE STRING "sirge simde is rack simde")
```

Comment:
little typo here btw, or is "sirge" the new name? :wink:

Reply:
Lol thanks!